guides/source/layout.html.erb

```diff
@@ -28,14 +28,22 @@
   <meta name="theme-color" content="#C81418">
 </head>
 
-<body dir="<%= @direction %>" class="guide">
+<body dir="<%= @direction %>" class="guide no-js">
+  <script>
+    document.body.classList.remove('no-js')
+  </script>
+
+  <a id="main-skip-link" href="#container" class="skip-link">
+    Skip to main content
+  </a>
+
   <nav id="topNav" aria-label="Secondary">
```

Comment:
It must have a label because it's not the primary navigation (otherwise it fails a wcag criterium), as there is already a primary navigation element.

I hate the label "secondary" personally, do you have a different suggestion?

Reply:
I understand that it's not the primary navigation, but when this element is shown, the element containing the primary navigation is hidden, so in theory this is the one that now becomes the primary navigation element. What do you think?

Reply:
I think you are right. Okay my proposal will be to drop the label and instead add an ERB comment for the next developer that the navs are mutually exclusive and thus not actually secondary.

Reply:
Is the name something used for voice navigation or is it more to provide context? Perhaps "global" and "chapter contents" or some other delineation that alludes to the content?

Reply:
In our screen readers it will do something along the lines of: "Secondary navigation". It must provide context. However, the "primary" (top) navigation is allowed to not have a contextual label. That's basically what we were discussing here.

Reply:
@brunoprietog I actually think this can be made just a `div`. It's not a `nav` in the sense that it only holds a single button that expands a set of links. Do you agree?